cmd/earthly/satellite_cmds.go

```diff
@@ -23,6 +24,16 @@ func (app *earthlyApp) satelliteCmds() []*cli.Command {
 			UsageText: "earthly satellite launch <satellite-name>\n" +
 				"	earthly satellite [--org <organization-name>] launch <satellite-name>",
 			Action: app.actionSatelliteLaunch,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
```

Comment:
You can use the `StringSliceFlag` type here. In which case I would call it `feature-flag` (singular).

You use it like this: `--feature-flag feat1 --feature-flag feat2 ...` on the command-line, and comma-separated via the env var.

Reply:
Oh that is cleaner, I'll switch to that